source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How the world taxonomy works
+parent: "/manual.html"
+layout: manual_layout
+section: Publishing
+type: learn
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2020-02-03
+review_in: 6 months
+related_applications: [content-tagger]
+---
+
+A World Taxon is an entity that GOV.UK content can be tagged to.  World taxons are represented on GOV.UK as pages such as [https://www.gov.uk/world/latvia] these are commonly referred to as world location pages. The full list of world location pages can be seen on https://www.gov.uk/world.
+
+There are other pages on GOV.UK under the /world taxonomy that are not these pages. There is more information on this in the Relationships section of this document and a full list of taxons [here][1].
+
+Whilst world taxons are treated in the same way as the regular [taxonomy][2] on a data level, they are a separate entity and do not share the same root. They cannot be found by visiting the root of the GOV.UK taxonomy meaning that they are simultaneously part of and not part of the core taxonomy.
+
+[1]: https://content-tagger.publishing.service.gov.uk/taxons/91b8ef20-74e7-4552-880c-50e6d73c2ff9
+[2]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+
+## Editing the Topic Taxonomy
+
+The world taxonomy is managed in [content-tagger][3]. Users must have the “GDS Editor” permission in content-tagger to see the relevant pages.
+
+The topics in the taxonomy (we call them “taxons” in code) are persisted in the publishing-api as content items. For an example [see the “Afghanistan” world taxon content item][4].
+
+This means that taxons inherit the publishing-api workflow, and can be in either draft state or published.
+
+[3]: https://content-tagger.publishing.service.gov.uk/taxons
+[4]: http://www.gov.uk/api/content/world/afghanistan
+
+## Root Taxon
+
+The taxon at the root of the world taxonomy (`/world`) is rendered by whitehall because there currently isn’t a template in collections that is suitable for rendering long lists without descriptions. This means that the actual root taxon as far as content-tagger is concerned is `/world/all` which is redirected to `/world`.
+
+## Relationships
+
+The link type `parent_taxons` is used to store the relationship between taxons. A [reverse][5] link called `child_taxons` is setup through the publishing-api.
+In order to avoid having a large number of near identical taxons world taxons are based on an associated taxon structure. There are a small number of generic taxons that should be displayed on each of the individual world pages, for example [passports and emergency travel documents][6]. These are then mirrored in the world pages, for example [passports and emergency travel documents Vietnam][7], which has information specific to that country.
```

Comment:
Maybe needs a comma, e.g. "In order to avoid having a large number of near identical taxons, world taxons are based on an associated taxon structure."

Reply:
done 👍